packages/server/migrations/20210826192131_access_tokens_add_uses.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable func-names */
+exports.up = function (knex) {
+    return knex.schema
+        .table('access_tokens', (table) => {
+            table.integer('uses').notNullable().defaultTo(0);
+        });
+};
+
+exports.down = function (knex) {
+    return knex.schema
+        .table('access_tokens', (table) => {
+            table.dropForeign('uses');
```

Comment:
knex is new to me; I don't understand why `dropForeign` is used here.

Reply:
You're right - it's an error. Thanks! Fix pushed.